src/components/ClientQuote/ClientQuote.module.scss

```diff
@@ -0,0 +1,46 @@
+.main {
+  display: flex;
+  flex-direction: row;
+
+  @media (max-width: 991px) {
+    flex-direction: column-reverse;
+  }
+}
+
+.imageWrapper {
+  position: relative;
+
+  @media (max-width: 991px) {
+    overflow: hidden;
+    max-height: 50vh;
+    height: 50vh;
+    max-width: 100vw;
+    width: 100vw;
+  }
+}
+
+.image {
+  height: 70vh;
+  width: 65vw;
+
+  @media (max-width: 991px) {
+    height: 50vh;
+    width: 105vw;
```

Comment:
what's the reason for having this expand past the viewport? we probably would be better off just cropping the image if youre trying to get specific positioning